.jenkins

```diff
@@ -399,6 +399,8 @@ pipeline {
                                 -DKokkos_ENABLE_TESTS=ON \
                                 -DKokkos_ENABLE_OPENMP=ON \
                                 -DKokkos_ENABLE_LIBDL=OFF \
+                                -DKokkos_ENABLE_LIBQUADMATH=ON \
+                                -DCMAKE_PREFIX_PATH=/usr/local/lib/gcc/x86_64-unknown-linux-gnu/5.3.0 \
```

Comment:
This is a bit frustrating.  This is only needed for CMake, `g++` does not need to add anything to the include path to find `<quadmath.h>`.  Also `-lquadmath` on the link line would do.